src/main/java/com/chikli/hudson/plugin/naginator/NaginatorPublisher.java

```diff
@@ -144,4 +167,101 @@ public Notifier newInstance(StaplerRequest req, JSONObject formData) throws Form
 
     private static final Logger LOGGER = Logger.getLogger(NaginatorPublisher.class.getName());
 
+    public static class Action extends NaginatorScheduleAction {
```

Comment:
Please call it `NaginatorAction`? I was really confused that `Action` comes from core